tools-local/tasks/Microsoft.DotNet.SourceBuild.Tasks/WriteBuildOutputProps.cs

```diff
@@ -55,6 +60,14 @@ public override bool Execute()
                 .OrderBy(id => id.Id)
                 .ToArray();
 
+            var additionalAssets = (AdditionalAssetPaths ?? new string[0])
+                .Where(Directory.Exists)
+                .Where(path => Directory.GetDirectories(path).Count() > 0)
+                .Select(path => new {
+                    Name = new DirectoryInfo(path).Name + "Version",
+                    Version = new DirectoryInfo(Directory.EnumerateDirectories(path).OrderByDescending(s => s).First()).Name
```

Comment:
`OrderBy(s => s).Last()` seems more intuitive for versions to me. (And it would match the above logic for package versions.)

Reply:
Fixed.